src/deployment/flavors.md

```diff
@@ -153,18 +153,24 @@ project 'Runner', {
 Setting up flavors in Android can be done in your project's 
 **build.gradle** file.
 
-1. In VSCode, inside your Flutter project, 
-navigate to **android**/**app**/**build.gradle**.     
+1. Inside your Flutter project, 
+   navigate to **android**/**app**/**build.gradle**.
 
-2. Add a flavor object with the specified environments along with values for 
-**dimension**, **resValue**, and **applicationId** or **applicationSuffix**. 
-  * The name of the application for each build is located in **resValue** 
-(as shown in the snippet below from VSCode).
-  * If you specify a **applicationSuffix** instead of a **applicationId**, 
-    it is appended to the "base" application id.
+2. Create a [`flavorDimension`][] to group any product flavors you will add
+   and prevent them from being combined.
```

Comment:
**chore:** Change the future tense to present tense per [GSG](https://developers.google.com/style/tense). 

**suggestion:** Separate the sentence at the `and`. You can say that the flavorDimension groups product flavors you add, then say this grouping prevents combining flavors.

Reply:
@atsansone You will have to look at the later commits or the total "Files changed". It now look likes the following:

```
Create a [`flavorDimension`][] to group your added product flavors.
Gradle doesn't combine product flavors that share the same `dimension`.
```